.gitlab-ci.yml

```diff
@@ -22,23 +22,23 @@ stages:
 
 build_wheel_linux_py27:
   <<: *build_linux
-  image: docker.apple.com/turi/build-image-centos6-python2.7:1801
+  image: registry.gitlab.com/coremltools/coremltools/build-image-ubuntu:1.0.0
```

Comment:
This is a really great fix. Thanks!! Let me try and test it out to make sure it works.